i18n/Cargo.toml

```diff
@@ -12,10 +12,16 @@ clap.workspace = true
 gettext-rs.workspace = true
 bytemuck = { version = "1.17.0", features = ["derive"] }
 byteorder = "1.5.0"
+strum = "0.26.3"
+strum_macros = "0.26.4"
```

Comment:
Delete the micro version of all deps.  e.g.
"0.26.3" -> "0.26"
"0.17.0" -> "0.17" 
etc.